CMakeLists.txt

```diff
@@ -120,7 +120,11 @@ elseif(CMAKE_COMPILER_IS_GNUCXX)
   endif(CYGWIN)
   set(CMAKE_CXX_FLAGS
     "${CMAKE_CXX_FLAGS} -Wall -pedantic -Werror -Wextra -Werror=shadow")
-  if (GCC_VERSION VERSION_GREATER 4.4)
+  if (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 4.4)
```

Comment:
`CMAKE_CXX_COMPILER_VERSION` can be anything - not just gcc, no?

Reply:
Yes, that's why there's a condition for gcc above.